src/node_contextify.cc

```diff
@@ -1038,38 +1069,43 @@ void ContextifyContext::CompileFunction(
     }
   }
 
-  MaybeLocal<Function> maybe_fun = ScriptCompiler::CompileFunctionInContext(
+  MaybeLocal<Function> maybe_fn = ScriptCompiler::CompileFunctionInContext(
       parsing_context, &source, params.size(), params.data(),
       context_extensions.size(), context_extensions.data(), options);
 
-  Local<Function> fun;
-  if (maybe_fun.IsEmpty() || !maybe_fun.ToLocal(&fun)) {
+  if (maybe_fn.IsEmpty()) {
     DecorateErrorStack(env, try_catch);
     try_catch.ReThrow();
     return;
   }
+  Local<Function> fn = maybe_fn.ToLocalChecked();
```

Comment:
Are you sure you want to do this? What if this assumption is false and this statement fails? Even if it doesn't, what's wrong about the way I originally intended to do it? I'm sorry but I am more cautious since such assumptions have kicked me in the balls in the past while working with the V8 API.